ansible/files/postgresql_config/postgresql.service.j2

```diff
@@ -19,6 +19,7 @@ TimeoutStartSec=86400
 Restart=always
 RestartSec=5
 OOMScoreAdjust=-1000
+EnvironmentFile=-/etc/environment.d/postgresql.env
```

Comment:
No particular concerns, but what gets included in this file and what's populating it? Or is it there just in case it's needed?

Reply:
@darora when nix installs the postgresql package, nix doesn't perform a lot of the operations that the debian package performs. So here I am adding the following values so that postgres will know about locales

this file only contains

LOCALE_ARCHIVE=/usr/lib/locale/locale-archive
LANG="en_US.UTF-8"
LANGUAGE="en_US.UTF-8"
LC_ALL="en_US.UTF-8"
LC_CTYPE="en_US.UTF-8"